packages/apollo/src/commands/service/check.ts

```diff
@@ -20,6 +21,7 @@ import {
 import moment from "moment";
 import sortBy from "lodash.sortby";
 import cli from "cli-ux";
+import { continueStatement } from "babel-types";
```

Comment:
Accidental import (which you've probably already removed somewhere)

Reply:
🤦‍♂